bare/roles/web/tasks/ruby.yml

```diff
@@ -3,66 +3,58 @@
 - name: Clone rbenv
   git:
     repo: "https://github.com/rbenv/rbenv.git"
-    dest: "{{ rbenv_home }}"
+    dest: "~/.rbenv"
```

Comment:
Not entirely sure, but shouldn't this be run as the `mastodon` user now that it's not longer global?

Reply:
Yes, it is run as `mastodon` user. Check the `main.yml` where I configure the user for this script. This prevents repeating of the user line in every item in this file.

Reply:
:+1: